src/clients/c++/request_http.cc

```diff
@@ -652,7 +652,7 @@ ModelControlHttpContextImpl::SendRequest(
   curl_easy_setopt(curl, CURLOPT_URL, full_url.c_str());
   curl_easy_setopt(curl, CURLOPT_USERAGENT, "libcurl-agent/1.0");
   // use POST method
-  curl_easy_setopt(curl, CURLOPT_POST, 1L);
+  curl_easy_setopt(curl, CURLOPT_POSTFIELDS, "");
```

Comment:
What does this change do?

Reply:
To provide an empty request body for POST request. Otherwise libcurl will use the default reader, which reads from stdin.

Reply:
ok